### PR TITLE
Creating Datamodel Tree

### PR DESCRIPTION
**Pull request by PaulHuwe**
Issue RCAL-33

### CODE REVIEW

**Comment by dmggh** [comment]
I'm not sure what I should be seeing here - a directory with 2 files of size 0 ?

**Comment by PaulHuwe** [comment]
Yeah, that's all it is.. Nadia asked me to put the structure in. If doing this alone is too simple, I can undo the PR and wait until I have files completed to do it.

**Comment by dmggh** [comment]
This may be a minor point, but in jwst there are :


jwst/datamodels/schemas/core.schema.yaml

jwst/datamodels/schemas/referencefile.schema.yaml


There is no:

jwst/datamodels/schemas/reference_files


Was there a decision to have all of the reference file schemas in their own directory ? I think it's probably a good idea.

________________________________

**Comment by nden** [comment]
@dmggh Yes, there was a decision to split the schemas for reference files in a separate directory. Also for jwst.
@PaulHuwe If you have the files ready add them to this PR. Add them even if they are not complete so we can look at them and comment.

**Comment by ddavis-stsci** [comment]
Here are the proposed initial metadata from INS for the flat ref file,
'Detector'; 'WFI01'
'Instrument':  'WFI'
'Description': 'Flat reference file'
'Author': 'Detector Lab'
'Date': ' '
'Useafter': ' '
'Origin':  'STScI'
'Filetype': 'ASDF'
'Reftype': 'Flat'
'Telescope': 'Roman'
'Pedigree': ''
'History': 'A novel is needed'
and these should be in the initial three schemas.

**Comment by PaulHuwe** [comment]
Initial core schema file with Nadia's suggestion's have been added. I will address Dave's comments in the references file.

**Comment by PaulHuwe** [comment]
I did not include filetype, as we are only working with ASDF now. I can put that back in, if desired.

File: romancal/datamodels/schemas/core.schema.yaml [hunk 1]
```diff
@@ -0,0 +1,109 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/fits-schema/fits-schema"
```
Comment (nden) [inline romancal/datamodels/schemas/core.schema.yaml:3]: This line is not necessary any more.

**Review by nden — COMMENTED** [review]
Looks good to me. Only need to remove 
`$schema: "http://stsci.edu/schemas/fits-schema/fits-schema"`

**Comment by nden** [comment]
I suggest we merge this now and include improvements later. This will allow periwinkle to use the datamodels subpackage for further development.

**Review by ddavis-stsci — APPROVED** [review]

**Comment by ddavis-stsci** [comment]
Merging PR

**Pull request merged**